File: RAI/Clase 5/java/Main.java

public class Main {
    public static void main(String[] args){
        //Creación de objetos
        //[Clase] [nombre del objeto]
        Gato oscar;
        
        //Instanciación
        //[nombre del objeto] = new [Clase](valores del objeto)
        oscar = new Gato("Óscar", "macho", 3, 7, "marrón", "rayado");

        System.out.println("El nuevo gato es: " + oscar.getName());

        Gato luna = new Gato("Luna", "hembra", 2, 5, "gris", "lisa");

        System.out.println("El nuevo gato es: " + luna.getName());

        oscar.eat();
        luna.move();
        oscar.meow();
        luna.purr();
        oscar.huntMice();

        System.out.println(luna.toString());

        //luna.color = "rosado";
        luna.changeColor("blanco");
        System.out.println("La gata " + luna.getName() + " es " + luna.getColor());
    }
}

File: RAI/Clase 5/python/Main.py
#from <nombre de archivo> import <nombre de clase>
from Gato import Gato

if __name__ == "__main__":
    #<nombre de objeto> = <nombre de clase>(atributos)
    oscar = Gato("Óscar", "macho", 3,7, "marrón", "rayado")
    #print("El nuevo gato es: " + oscar.__name)

    luna = Gato("Luna", "hembra", 2,5,"gris", "lisa")

    #print("El nuevo gato es: " + luna.__name)

    oscar.eat()
    luna.move()
    oscar.meow()
    luna.purr()
    oscar.hunt_mice()

    print(luna)

    #luna.__color = "rosado"

    #print("La gata " + luna.__name + " es " + luna.__color)